python/doc/examples/reliability_sensitivity/sensitivity_analysis/plot_sensitivity_wingweight.py

```diff
@@ -0,0 +1,462 @@
+"""
+Example of sensitivity analyses on the wing weight model
+=========================================================
+"""
+import openturns as ot
+import openturns.viewer as otv
+from openturns.usecases.wingweight_function import WingWeightModel
+from matplotlib import pylab as plt
+import numpy as np
+ot.Log.Show(ot.Log.NONE)
+
+# %%
+#
+# This example is a brief overview of the use of main sensitivity analysis techniques and how to call them.
+# In this example, the following sensitivity classes are used:
+#
+# - PCC: Partial Correlation Coefficients
+# - PRCC: Partial Rank Correlation Coefficients 
+# - SRC: Standard Regression Coefficients
+# - SRRC: Standard Rank Regression Coefficients
+# - Pearson coefficients
+# - Spearman coefficients
+# - Taylor expansion importance factors
+# - Sobol' indices
+# - HSIC : Hilbert-Schmidt Independence Criterion
+#
+# We present the methods on the :ref:`WingWeight function<use-case-wingweight>`.
+
+# %%
+# Definition of the model
+# -----------------------
+#
+# We load the model from the usecases module.
+# 
+# For a quick reminder, this usecase is a ten dimensional model, with the following inputs:
+#
+# - :math:`S_w`: Wing area (ft^2), defined according to :math:`\mathcal{U}(150, 200)`
+# - :math:`W_{fw}`: Weight of fuel in the wing (lb), defined according to :math:`\mathcal{U}(220, 300)`
+# - :math:`A`: Aspect ratio (-), defined according to :math:`\mathcal{U}(6, 10)`
+# - :math:`\Lambda`: Quarter chord sweep (deg), defined according to :math:`\mathcal{U}(-10, 10)`
+# - :math:`q`: Dynamic pressure at cruise (lb/ft^2), defined according to :math:`\mathcal{U}(16, 45)`
+# - :math:`l`: Taper ratio (-), defined according to :math:`\mathcal{U}(0.5, 1)`
+# - :math:`t_c`: Airfoil thickness to chord ratio (-), defined according to :math:`\mathcal{U}(0.08, 0.18)`
+# - :math:`N_z`: Ultimate load factor (-), defined according to :math:`\mathcal{U}(2.5, 6)`
+# - :math:`W_{dg}`: Flight design gross weight (lb), defined according to :math:`\mathcal{U}(1700, 2500)`
+# - :math:`W_p`: Paint weight (lb/ft^2), defined according to :math:`\mathcal{U}(0.025, 0.08)`
+m = WingWeightModel()
+
+# %%
+# Cross cuts of the function
+# --------------------------
+#
+# Let's have a look on 2D cross cuts of the wing weight function.
+# For each 2D cross cut, the other variables are fixed to the input distribution mean values.
+# This graph allows to have a first idea of the variations of the function in pair of dimensions.
+# The colors of each contour plot are comparable. The number of contour levels are related to the amount of variation of the function in the corresponding coordinates.

```

Comment:
These graphs are great. I think it would be worthwhile to develop services in the `Function` classes to produce such graphs natively.